GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java

```diff
@@ -743,13 +799,21 @@ private void loadTexture(GVRAssetLoader.AssetRequest assetRequest,
         }
         if (texIndex > 0)
         {
+            if (usingPBR)
+            {
+                assetRequest.onTextureError(mContext, textureKey + " duplicate ignored for PBR renderer" + mFileName, mFileName);
```

Comment:
This is why we have the test for PBR. The GLTF2 importer generates two diffuse maps in that specific case and we ignore one. In all other cases, we don't ignore that map. We have to call the error callback so the texture is "counted" by the asset loader. Otherwise it will be waiting for that texture to be loaded.

I think a future version of jassimp may handle this differently because I looked at their newer source code. I could try pulling out the newest GLTF 2 importer from jassimp and see if it works with the rest of 4.1.